src/main/java/org/valdroz/vscript/BaseNode.java

```diff
@@ -341,17 +341,32 @@ public Variant execute(VariantContainer variantContainer) {
                 }
                 LocalVariantContainer lvc = new LocalVariantContainer(variantContainer);
                 List<String> parameterNames = function.getParameterNames();
-                for (int pidx = 0;
-                     pidx < Math.min(params.size(), parameterNames.size());
-                     ++pidx) {
-                    String fpn = parameterNames.get(pidx);
-                    if (fpn.length() > 0) {
-                        lvc.setVariant(fpn, params.get(pidx).execute(variantContainer));
+                if(getName().equalsIgnoreCase(NT_FUNCTION_IF)) {
```

Comment:
1) Code formatting issueL Missing space after `if`:
2) Can a handling for function `if` be implemented as separate `case NT_MF_IF`?

Reply:
introduced `NT_MF_IF` implementation